mapper/mapper.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/ONSdigital/dp-api-clients-go/zebedee"
 	"github.com/ONSdigital/dp-cookies/cookies"
 	"github.com/ONSdigital/dp-frontend-dataset-controller/helpers"
+	commonModels "github.com/ONSdigital/dp-frontend-dataset-controller/model"
```

Comment:
Not sure on the pluralisation of the model here and I think having a name liked `shared` would give future devs the understanding that if they were to remove or change a prop it could have implications across multiple uses of the model.
```suggestion
	sharedModel "github.com/ONSdigital/dp-frontend-dataset-controller/model"
```

Reply:
> The variables table looks good, as per design, I would consider using the [responsive table variant](https://ons-design-system.netlify.app/components/table/#responsive-table); users in mobiles will find the current implementation clunky on their devices.
> 
> ![image](https://user-images.githubusercontent.com/19624419/135484684-d2c3ccc8-bc6d-400f-8cb4-c99a79b5d10a.png)

Updated 
![image](https://user-images.githubusercontent.com/23668262/135586242-1af97ca8-c9b5-48f7-979a-89a07ed475df.png)